vavr/src/main/java/io/vavr/control/Validation.java

```diff
@@ -91,24 +91,32 @@
     }
 
     /**
-     * Creates an {@link Invalid} that contains the given {@code error}.
+     * Creates an {@link Invalid} that contains the given {@code errors}.
      *
-     * @param <E>   type of the given {@code error}
+     * @param <E>   type of the given {@code errors}
      * @param <T>   type of the value
-     * @param error An error
-     * @return {@code Invalid(error)}
-     * @throws NullPointerException if error is null
+     * @param error An errors
+     * @return {@code Invalid(errors)}
+     * @throws NullPointerException if errors is null
      */
     static <E, T> Validation<E, T> invalid(E error) {
-        Objects.requireNonNull(error, "error is null");
+        Objects.requireNonNull(error, "errors is null");
         return new Invalid<>(error);
     }
 
+    static <E, T> Validation<E, T> invalid(Seq<? extends E> errors) {
+        Objects.requireNonNull(errors, "errors is null");
+        if (errors.size() == 0) {
+            throw new IllegalArgumentException("Errors are empty");
+        }
+        return new Invalid<E, T>(errors);
+    }
+
     /**
      * Creates a {@code Validation} of an {@code Either}.
      *
      * @param either An {@code Either}
-     * @param <E>    type of the given {@code error}
+     * @param <E>    type of the given {@code errors}
```

Comment:
I think this was a copy-paste error by us. Better would be to name it

```java
* @param <E>    error type
```

or (like below)

```java
* @param <E>    type of errors
```